cpp/include/cugraph/detail/graph_utils.cuh

```diff
@@ -47,6 +48,22 @@ struct compute_gpu_id_from_vertex_t {
   }
 };
 
+template <typename vertex_t>
+struct compute_gpu_id_from_int_vertex_t {
```

Comment:
Should we better rename other functors working on external vertex IDs to `ext_vertex_t` and `ext_edge_t`?

Reply:
Done for vertex in the next push.

Do we ever try and use these functors on an `int_edge_t`?  I'm inclined not to add the `ext` to the name unless we need to distinguish.

Reply:
Gotcha, agreed.